### ISSUE TITLE
Clearing filters returns Uncaught TypeError in Chrome

### ISSUE THREAD

**Issue by jchafik** [vedmack/yadcf#220]
Attempting to clear filters in Chrome returns `Uncaught TypeError: Cannot read property 'length' of undefined`.

Reason is that the condition check below is incorrect. It should read `if (typeof settingsDt.aanFeatures.f !== 'undefined') {...}`
https://github.com/vedmack/yadcf/blob/master/jquery.dataTables.yadcf.js#L3731

**Comment by vedmack** [comment]
please provide a link that reproduce this issue, because on the showcase it works just fine

**Comment by jchafik** [comment]
Is settingsDt.aanFeatures.f undefined in the example you're looking at? I'll try to get something up, meanwhile, you can test out the following in chrome case for yourself.

```
var item = {};
if (item.feature == 'undefined') {
  console.log("This should, but will not execute.");
}
```

**Comment by jchafik** [comment]
http://jsfiddle.net/0y350vja/

**Comment by vedmack** [comment]
fixed in [0.8.9.beta.10](https://github.com/vedmack/yadcf/blob/405ccded2e8f026866420c3f429386850dc6ad0e/jquery.dataTables.yadcf.js)
changed into `settingsDt.aanFeatures.f !== undefined` the original line of code was copied from some sample I found around without noticing that the undefiend was a string
